### PR TITLE
Update README and Makefile

### PR DESCRIPTION
**Pull request by kirawoods**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -4,13 +4,13 @@ SHELL := /bin/bash
 # Setup
 ################################
 
-setup: setup-python setup-poetry install-deps setup-pre-commit setup-node setup-fern
+setup: setup-python setup-poetry install-deps setup-pre-commit setup-fern
```
Comment (noanflaherty) [inline Makefile:7]: @dvargas92495 for context, including `setup-node` produced errors for both Kira and I, so we removed for now. Kira, if you still have the console output handy with that error, could you show here?

File: ee/codegen/README.md [hunk 2]
```diff
@@ -39,11 +39,15 @@ To update the table of contents, run `doctoc README.md`
    ```bash
    npm install
    ```
+5. If you haven't already, go to the CONTRIBUTING.md guide and run the setup steps there
```
Comment (noanflaherty) [inline ee/codegen/README.md]:
```suggestion
5. If you haven't already, go to the [CONTRIBUTING.md](/CONTRIBUTING.md) guide and run the setup steps there
```

File: ee/codegen/README.md [hunk 3]
```diff
@@ -39,11 +39,15 @@ To update the table of contents, run `doctoc README.md`
...
 
 ## Testing
 
 To run unit tests:
 
+For these tests to run successfully, you'll need to run the poetry virtual environment.
+The steps to set this up are in the README at the root of this repo.
```
Comment (noanflaherty) [inline ee/codegen/README.md]:
```suggestion
For these tests to run successfully, you'll need to have an active python virtual environment with `isort` installed.
The steps to set this up are in the [CONTRIBUTING.md](/CONTRIBUTING.md) guide at the root of this repo.
```

**Review by noanflaherty — APPROVED** [review]
Let's please avoid committing changes to our package lock files

File: Makefile (hunk 1 above)
Comment (kirawoods) [reply Makefile:7]: I don't have the console output for that anymore, sorry!

Comment (dvargasfuertes) [reply Makefile:7]: do we remember what the general vibe of the console output was?

Comment (kirawoods) [reply Makefile:7]: Something to do with nvm I think?

**Pull request merged**